app/controllers/static_controller.rb

```diff
@@ -0,0 +1,6 @@
+class StaticController < ApplicationController
+
+  def xml
+  end
+  
```

Comment:
goal is to have /xml that displays the XML for the app so tool consumer can just add the URL. Would love some help on how to display XML in rails. my 
```
respond_to do |format|
format.xml
end 
```

did not work out